src/modules/gameAPI.js

```diff
@@ -0,0 +1,45 @@
+import { nameInput, scoreInput, submitScore } from './domElements.js';
+
+const myGameId = 'L0iNoCmhdpl';
+const baseAPIUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const scoresEndpoint = `${`${baseAPIUrl}games/${myGameId}/scores/`}`;
+
+class Score {
+  constructor(name, score) {
+    this.user = name;
+    this.score = score;
+  }
+}
+
+const getGameScores = async () => {
+  const response = await fetch(scoresEndpoint);
+  const data = await response.json();
+  const myResponses = data.result;
+  return myResponses;
+};
```

Comment:
- [ ] You have done great so far :+1: :+1:  But there are some changes you need to make before moving to the next milestone. :+1: 

- [ ] Your scores are added to the API correctly, then you have a function `getGameScores` that's meant to fetch the score from the API, but it seems you are not calling this function anywhere. Looking at the video below, you can see that it can add a score and refresh to see it, but once I reload the page, the refresh button doesn't fetch the scores. it seems that the refresh bottom only displays the scores stored locally. I'll suggest you work on fetching the scores from the API and showing them on the page. :+1: :smile: 

- - VIDEO:

https://github.com/leloufadel/-Leaderboard/assets/107632780/f96ca8ba-d4be-4bd3-8180-93df160a7c05